pymongo/write_concern.py

```diff
@@ -20,14 +20,6 @@
 from pymongo.errors import ConfigurationError
 
 
-# Moved here to avoid a circular import.
-def validate_boolean(option: str, value: Any) -> bool:
```

Comment:
Why change this?

Reply:
Otherwise importing it from `pymongo.common` raises an `attr-defined` error, unless we added an `__all__` there.

Reply:
We can't make this change because validate_boolean is public: https://pymongo.readthedocs.io/en/latest/api/pymongo/write_concern.html#pymongo.write_concern.validate_boolean

Reply:
Okay, I'll go with `__all__` then.

Reply:
Actually, it is a trivial function, we can just duplicate it.

Reply:
Let's not duplicate it since that will be confusing later. Instead can we never import it from common and only ever import it from write_concern?

Reply:
Done.